solver/src/driver.rs

```diff
@@ -79,6 +85,24 @@ impl Driver {
             .chain(amms.into_iter().map(Liquidity::Amm))
             .collect();
 
+        /*
```

Comment:
If you want to merge this PR (I can see auto-merge is already enabled) can you please remove this code?